backend/ledgers.ts

```diff
@@ -16,10 +18,17 @@ interface LedgerSql {
   operation_count: string;
 }
 
-let cachedData: Array<Ledger>;
-
-export const handler = function (_: any, res: Response) {
-  res.send(cachedData);
+export const handler = async function (
+  _: any,
+  res: Response,
+  next: NextFunction,
+) {
+  let cached = await redisClient.get("ledgers");
```

Comment:
since this var isn't reassigned, you should use `const` here. 

I see that we don't have eslint setup for this repo yet. Our @stellar/eslint-config will pick up this error for you. No need to implement eslint on this PR as it will likely complain about a lot of nit picky stuff, but probably a good idea to implement sooner rather than later. 

Example: https://github.com/stellar/account-viewer-v2/blob/master/.eslintrc.js

Reply:
https://github.com/stellar/dashboard/pull/205/commits/15aa33be37824ac00766761c729086d0c061f355

good idea, I'll add it next, tracked here: https://github.com/stellar/dashboard/issues/207